src/components/TrustedCompanies/TrustedCompanies.jsx

```diff
@@ -0,0 +1,141 @@
+import {React, useState, useRef, useEffect } from 'react';
+import { Box, Typography, ImageList, ImageListItem, useMediaQuery, makeStyles } from '@material-ui/core';
+import { useTheme } from '@material-ui/core/styles';
+import ArrowForwardIosIcon from '@material-ui/icons/ArrowForwardIos';
+import ArrowBackIosIcon from '@material-ui/icons/ArrowBackIos';
+
+import Image from '../../common/Image';
+
+
+
+import twilio from '../../assets/svg/twilio.svg';
+import disney from '../../assets/svg/disney.svg';
+import flipkart from '../../assets/svg/flipkart.svg';
+import intel from '../../assets/svg/intel.svg';
+import microsoft from '../../assets/svg/microsoft.svg';
+import ola from '../../assets/svg/ola.svg';
+import oracle from '../../assets/svg/oracle.svg';
+import paypal from '../../assets/svg/paypal.svg';
+import paytm from '../../assets/svg/paytm.svg';
+import phonepe from '../../assets/svg/phonepe.svg';
+import walmart from '../../assets/svg/walmart.svg';
+import uber from '../../assets/svg/uber.svg';
+
+
+
+
+
+const useStyles = makeStyles((theme) => ({
+    root: {
+        flexGrow: 1,
+        textAlign: 'center',
+    }, 
+    mainContainer: {
+        display: 'flex',
+        flexDirection: 'row',
+        flexWrap: 'wrap',
+        overflow: 'hidden',
+        backgroundColor: theme.palette.background.paper,
+    },
+    imageList: {
+        flexWrap: 'nowrap',
+        // Promote the list into his own layer on Chrome. This cost memory but helps keeping high FPS.
+        transform: 'translateZ(0)',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/n4alphapesto/hirebook-frontend/pullRequest?prid=7756469)
```suggestion
        transform: "translateZ(0)",
```